### PR TITLE
Basic Windows Support

### PR DESCRIPTION
**Pull request by LogoiLab**
A simple Cargo.toml edit and some additions of some compiler logic. In reference to Issue #44

### CODE REVIEW

**Comment by LogoiLab** [comment]
successfully builds on windows 10 rust nightly and stable

File: src/clip.rs [hunk 1]
```diff
@@ -29,10 +29,9 @@ pub fn copy_to_clipboard(s: &SafeString) -> Result<(), ()> {
 
 // On UNIX, the most stable way to copy to the clipboard is using one of the existing
 // and battle tested tools: xsel and xclip.
-#[cfg(all(unix, not(macos)))]
+#[cfg(all(unix, not(macos), not(windows)))]
```
Comment (conradkdotcom) [inline src/clip.rs]: What does this do ? Given that I have selected `unix` and Windows is not a UNIX, I feel like this change is useless. What do you think ?

File: src/commands/mod.rs [hunk 2]
```diff
@@ -24,6 +24,9 @@ pub mod set_master_password;
 pub mod rename;
 pub mod transfer;
 pub mod change;
+
```
Comment (conradkdotcom) [inline src/commands/mod.rs]: Please remove this space

File: src/commands/mod.rs [hunk 3]
```diff
@@ -24,6 +24,9 @@ pub mod set_master_password;
...
+#[cfg(not(target_os = "windows"))]
 pub mod uninstall;
+
```
Comment (conradkdotcom) [inline src/commands/mod.rs]: Please remove this space

File: src/main.rs [hunk 4]
```diff
@@ -20,7 +20,10 @@ extern crate crypto;
 extern crate rpassword;
 extern crate rand;
 extern crate byteorder;
+
```
Comment (conradkdotcom) [inline src/main.rs]: Please remove this space

File: src/main.rs [hunk 5]
```diff
@@ -20,7 +20,10 @@ extern crate crypto;
...
+#[cfg(not(target_os = "windows"))]
 extern crate quale;
+
```
Comment (conradkdotcom) [inline src/main.rs]: Please remove this space

File: src/clip.rs [hunk 6]
```diff
@@ -29,10 +29,9 @@ pub fn copy_to_clipboard(s: &SafeString) -> Result<(), ()> {
...
 pub fn copy_to_clipboard(s: &SafeString) -> Result<(), ()> {
     use shell_escape;
-    use quale::which;
```
Comment (conradkdotcom) [inline src/clip.rs]: This is useless, given the above comment, and breaks the UNIX builds (Linux, Mac OSX). Please revert this change, unless this is useful.

File: src/clip.rs (hunk 1 above)
Comment (HackU6-Goats) [reply src/clip.rs]: For whatever reason cargo tries to compile that function unless I specifically tell it not too.

File: src/clip.rs (hunk 6 above)
Comment (HackU6-Goats) [reply src/clip.rs]: Crap that shouldn't have been there. I'll fix that.

File: src/commands/mod.rs (hunk 2 above)
Comment (HackU6-Goats) [reply src/commands/mod.rs]: Sure thing. It's there for readability sale though

File: src/commands/mod.rs (hunk 3 above)
Comment (HackU6-Goats) [reply src/commands/mod.rs]: Ditto

File: src/main.rs (hunk 4 above)
Comment (HackU6-Goats) [reply src/main.rs]: Ditto

File: src/main.rs (hunk 5 above)
Comment (HackU6-Goats) [reply src/main.rs]: Ditto

**Comment by LogoiLab** [comment]
Just realized I was on a temp account from a hackathon yesterday.

**Comment by LogoiLab** [comment]
I think I'm gonna go ahead and close this and come back with a new one.

File: src/commands/mod.rs (hunk 2 above)
Comment (conradkdotcom) [reply src/commands/mod.rs]: @HackU6-Goats Oh, alright, well then feel free to let it there. Or best, if you can, run `cargo fmt` before committing, so we have standard styling everywhere :)

File: src/clip.rs (hunk 1 above)
Comment (conradkdotcom) [reply src/clip.rs]: Hmm, alright, odd, well then feel free to let this here, I don't mind, just seems odd

File: src/clip.rs (hunk 6 above)
Comment (conradkdotcom) [reply src/clip.rs]: Alright, thanks a lot ! :)

**Comment by conradkdotcom** [comment]
@LogoiLab Thanks a lot for your replies. Feel free to open an new PR when you wish and I'll review ASAP :) Thanks again for your help !
